sklearn/datasets/_openml.py

```diff
@@ -850,6 +850,10 @@ def fetch_openml(
       typically loaded as such `(0, 1, ...)` with the `"pandas"` parser while
       `"liac-arff"` will force the use of string encoded class labels such as
       `"0"`, `"1"` and so on.
+    - The `"pandas"` parser will not strip single quotes - i.e. `'` - from
```

Comment:
I don't think there will be datasets in parquet on openml where 'single quoted string' values are meant to be silently unquoted. Otherwise those quotes would not be stored in the parquet fields in the first place. parquet is a binary format where field, it does not rely on quotes as a delimiter at all, so there is no should be no need for stripping anything.